src/main/java/nl/novi/TechItEasy/services/TelevisionService.java

```diff
@@ -0,0 +1,98 @@
+package nl.novi.TechItEasy.services;
+
+import nl.novi.TechItEasy.dtos.TelevisionOutputDto;
+import nl.novi.TechItEasy.dtos.TelevisionInputDto;
+import nl.novi.TechItEasy.models.Television;
+import nl.novi.TechItEasy.repositories.TelevisionRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class TelevisionService {
+
+
+    private final TelevisionRepository televisionRepository;
+    public TelevisionService(TelevisionRepository televisionRepository) {
+        this.televisionRepository=televisionRepository;
+    }
+
+    public List<TelevisionOutputDto> getAllTelevisions() {
+        return televisionRepository.findAll()
+                .stream()
+                .map(TelevisionService::toTelevisionDto)
+                .collect(Collectors.toList());
+    }
+
+    public TelevisionOutputDto getTelevisionById(int id) {
+        Television t = televisionRepository.findById(id).get();
```

Comment:
Je doet hier een .get() zonder te checken of er wel werkelijk iets in de Optional zit. 
Ik weet niet precies wat er dan gebeurt. Misschien dat jou applicatie dan meteen al crasht of misschien dat jou "Television t" dan null wordt en je applicatie in de teTelevisionDto methode crashed wanneer de code daar "null.getName()" probeert uit te voeren.